### DISCUSSION TITLE
Is there a method to customize scopes

### DISCUSSION THREAD

**Discussion by coinlet** [casdoor/casdoor#3090] [Q&A] [answered]
Is there a method to customize scopes? Some services require specific scopes when using Casdoor as the OIDC provider. For example, Homarr needs to provide a scope named "groups."

[https://homarr.dev/docs/advanced/sso/](https://homarr.dev/docs/advanced/sso/)

How should I configure ?

thanks.

**Comment by rongchuan17** [comment]
Have you solved this problem? I'm also having the same issue.

**Reply by hsluoyz** [reply, marked as answer]
Replaced by: https://github.com/casdoor/casdoor/issues/3617
